x-pack/plugins/observability_solution/apm/server/routes/assistant_functions/get_observability_alert_details_context/get_downstream_dependency_name.ts

```diff
@@ -50,9 +49,10 @@ export async function getDownstreamServiceResource({
           ],
         },
       },
+      fields: ['span.destination.service'],
     },
   });
 
   const hit = response.hits.hits[0];
-  return hit?._source?.span.destination?.service.resource;
+  return hit?.fields?.['span.destination?.service?.resource']?.[0] as string;
```

Comment:
question marks here in the string need to be removed and the type should be `string | undefined` presumably?